lib/framework/persistence/repositories/InMemoryLogRepository.js

```diff
@@ -25,10 +25,40 @@ class InMemoryLogRepository extends LogRepository {
     /**
      * Returns all entities.
      *
-     * @returns {Promise} Promise object represents the ...
+     * @returns {Promise} Promise object representing the full mock data
      */
     async findAll() {
-        return Promise.resolve([]);
+        const date = new Date().toDateString();
```

Comment:
Date should be epoch/unix (or some other format) and the appropriate timezone formatting should be done on the frontend.

Reply:
@mvegter Is it not better to wait until we know exactly how we will be storing dates in our future database? Otherwise we will constantly be refactoring.

Reply:
@MauritsioRK No, as the frontend is responsible for the appropriate time zone formatting not the backend regardless of database implementation.

Reply:
Unless the decision is already made somewhere, you simply can't be 100% certain that we will be using unix to store our dates. There is always a chance that something is incompatible.

While I do agree with your point, I feel that it falls outside of the scope of this PR/ticket. A date formatter is something to be implemented globally. Remember that this is the ticket for the detail view UI.

Reply:
@MauritsioRK that decision has not yet been made. By that same logic how do you know which columns to use/implement?

Yes this is the ticket for the UI, but my logic the UI is responsible for displaying the data thus date formatting is also part of it.

Reply:
The columns were inspired from the documentation found in, https://github.com/SoftwareForScience/jiskefet-miscellaneous-and-discussion/blob/master/Metadata-for-detector-commissioning.md as mentioned by Adam and George in the workshop at the start of the project.

![image](https://user-images.githubusercontent.com/14125058/80135726-e915c100-85a0-11ea-8cec-b587b9d1c20f.png)

Reply:
@john-ghatas the point is not whether or not you know the required columns. Every single aspect of this project is subject to change as the requirements are not clear nor complete (required for v1 implies that the view will be changed for example).